modules/acquia_cms_headless/acquia_cms_headless.info.yml

```diff
@@ -10,3 +10,5 @@ dependencies:
   - drupal:openapi_jsonapi
   - drupal:openapi_ui
   - drupal:openapi_ui_redoc
+  - drupal:rest
+  - drupal:restui
```

Comment:
Also missing consumers and simple_oauth modules.

Reply:
added consumers and simple_oauth dependencies.

Reply:
I think that's ok, even if we don't add this modules explicitly. Because next module will enable them first, as it has dependency over those (i.e. consumers and simple_oauth ) modules.